Tests/mocks/devicesettings.h

```diff
@@ -1215,6 +1228,399 @@ class FrontPanelConfig {
         return impl->getTextDisplay();
     }
 };
-
 }
 
+
+typedef uint32_t Op_t;
+class Exception : public std::exception
+{
+	public:
+		virtual const char* what() const throw()
+		{
+			return "Base Exception..";
+		}
+
+};
+class CECNoAckException : public Exception
+{
+	public:
+		virtual const char* what() const throw()
+		{
+			return "Acl not received..";
+		}
+};
+class IOException : public Exception
+{
+	public:
+		virtual const char* what() const throw()
+		{
+			return "IO Exception..";
+		}
+};
+
+class CECFrameImpl{
+	public:
+		virtual void getBuffer(const uint8_t **buf, size_t *len) const = 0;
+
+};
+class CECFrame{
+	public:
+	
+	CECFrameImpl* impl;
+	void getBuffer(const uint8_t **buf, size_t *len) const{
+		return impl->getBuffer(buf, len);
+	}
+	CECFrame(const uint8_t *buf = NULL, uint16_t len = 0){}
+
+};
+class CECBytesImpl {
+	public:
+	virtual const std::string toString() const = 0;
+};
+class CECBytes{
+    public:
+    CECBytesImpl* impl;
+    static CECBytes& getInstance()
+    {
+        static CECBytes instance;
+        return instance;
+	}
+    const std::string toString(void) const {
+		return getInstance().impl->toString();
+    }
+};
+
+class LogicalAddressImpl{
+	public:
+		virtual int toInt() const = 0;
+		virtual int getType() const = 0;
+        virtual std::string toString() const = 0;
+
+};
+class LogicalAddress : public CECBytes{
+	public: 		
+		static const LogicalAddress kTv;
```

Comment:
how is kTv used?

Reply:
It doesn't seem to be needed upon an initial look. Will remove, and make sure it's not actually needed.